Readme.md

```diff
@@ -3,13 +3,16 @@
 [![npm version](https://badge.fury.io/js/eswrap.svg)](https://badge.fury.io/js/eswrap)
 [![Build Status](https://travis-ci.org/kevinastone/eswrap.svg?branch=master)](https://travis-ci.org/kevinastone/eswrap)
 
-eswrap is a ecmascript formatting utlity.  It will parse a supplied block of code and apply formatting to attempt to fit it within a col width.
+**eswrap** is an ECMAScript formatting utility.
+It attempts to reformat a code block to fit within a specified column count:
 
-```
+```bash
 echo "[variable1, variable2, variable3];" | eswrap --limit 20
-[
+
+# Output:
+'[
   variable1,
   variable2,
   variable3,
-];
+];'
```

Comment:
Yeah, on second thought, it was pretty dumb of me to use that as a hack to control highlighting colour (and hell, the braces and semicolon aren't [coloured on GitHub anyway](https://github.com/Cutlery-Drawer/eswrap/blob/fix-typos/Readme.md)). Sorry.

Fixed!